### ISSUE TITLE
[a11y] Cannot create & add new record to record set & database

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1264]
**Describe the bug**
When viewing a record set, I cannot create a new record and add it to the set & database.

**To Reproduce**
1. Go to a record set
2. Click **+**
3. See specify loading logo indefinitely

**Expected behavior**
In current version, 
![image](https://user-images.githubusercontent.com/37256050/163903465-0b8f2d5f-2f27-460d-b57d-f9010123cbd1.png)

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/163903356-ce5dd558-41c5-42e9-bfc6-c612e3cc6fc8.png)

macOS & Edge

**Comment by maxxxxxdlp** [comment]
Fixed in 91663c55
